buildconfig/Jenkins/jenkins-agent.sh

```diff
@@ -88,6 +92,14 @@ if [ ! -z "${PROXY_HOST}" ]; then
   export https_proxy=https://$PROXY_HOST:$PROXY_PORT
 fi
 
+if [[ "$OSTYPE" == "darwin"* ]] && [[ $(uname -m) == 'arm64' ]]; then
+  JAR_FILE=$MAIN_JAR_FILE
```

Comment:
Does it make sense to call this the main jar file when only 1/4 configurations of runner uses it?

`ARM_JAR_FILE`?